jenkins/scripts/fetch_logs.sh

```diff
@@ -63,6 +64,15 @@ if [[ "${BARE_METAL_LAB}" == true ]]; then
         -i "${METAL3_CI_USER_KEY}"
         -o ProxyCommand="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i ${METAL3_CI_USER_KEY} -W %h:%p ${METAL3_CI_USER}@${JUMPHOST_IP}"
     )
+elif [[ -n "${SSH_JUMP_HOST}" ]]; then
+    declare -a SSH_OPTIONS=(
+        -o StrictHostKeyChecking=no
```

Comment:
We are duplicating a lot of these options. Maybe declare the array with before the `if` with the common options and only add the case specific ones inside the `if`